### ISSUE TITLE
[FEEDBACK]: Missing instruction in ToDo app tutorial

### ISSUE THREAD

**Issue by sgmtbd** [TBD54566975/developer.tbd.website#933]
### Type

Docs Bug Report

### Feedback

Tell us what you're seeing!
In this snippet

`
git clone https://github.com/TBD54566975/developer.tbd.website.git
pnpm install
pnpm start:tutorial-todo-starter
`

there is a missing instruction. User needs to `cd developer.tbd.website` between the first and second steps.

### Screenshots

Paste relevant images or screenshots here
<img width="598" alt="image" src="https://github.com/TBD54566975/developer.tbd.website/assets/145713320/44838394-b3bb-456f-89d8-e90ffb15110e">

### developer.tbd.website URL

https://developer.tbd.website/docs/web5/build/apps/todo-app-tutorial

**Comment by sgmtbd** [comment]
I'm finding a lot of small typos/errors (unless I'm missing something). Might be worth going through the tutorial from start to finish and correct.

**Comment by EbonyLouis** [comment]
Thank you so much for calling these out. There are some pending edits.

**Comment by blackgirlbytes** [comment]
Hey Sanjay and Ebony! For clarity, this tutorial doesn't require you to cd into the project. Bobbilee and Nick set this tutorial up so that you can run pnpm start to-do-project from the root level and it will immediately start working. 

And as for small typos/errors, we can probably open this up to the community because this is the todo app v1 ..and not the one Ace built. (Unless he does want to work on it, but wanted to give him the freedom to opt-out since I saw he was tagged)

**Comment by EbonyLouis** [comment]
@blackgirlbytes Thanks for catching that Rizel. I didn't fully read through it and assumed it was the new one. I'm still sick and taking today off. I'll create some issues tomorrow.

**Comment by acekyd** [comment]
I'm picking this up and would do a quick sweep. The original post is correct, we were missing a step to be able to do the pnpm install.

**Comment by blackgirlbytes** [comment]
@acekyd, it works without doing `cd`
This line is in the package.json
`"start:tutorial-todo-starter": "pnpm --filter todo-starter run dev"`
The reason you don't have to cd into the directory of the "todo-starter" package is because of the --filter flag. It allows you to target specific packages in a monorepo from the root directory without having to navigate into each package's directory.

Below screenshot of it working for me.
<img width="642" alt="Screenshot 2023-10-27 at 8 51 16 AM" src="https://github.com/TBD54566975/developer.tbd.website/assets/22990146/2a7f08ed-b5bc-4624-b248-a47a8f0f62d5">

Either way, adding the `cd` doesn't hurt the tutorial, so that's fine.

**Comment by acekyd** [comment]
Yes @blackgirlbytes but when you clone for the first time, you'll be outside the folder you clone. It's intuitive to cd into `developer.tbd.website` but the steps we mentioned skips that. The screenshot you shared shows you're already in the `developer.tbd.website` folder.

**Comment by blackgirlbytes** [comment]
ah okay. that's true!

**Comment by sgmtbd** [comment]
Yep, what Adewale said. :) I agree with Rizel that it's intuitive to many
people, but not to everyone. :) Thanks, everyone! You seem like such a good
team that works (and plays!) so well together. I'm jealous! :)

Thanks,
Sanjay
